src/components/SidebarLayout/Sidebar/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Divider, IconText } from '@gnosis.pm/safe-react-components'
+
+import List, { ListItemType } from '../../List'
```

Comment:
The only place you forgot to use absolute routes!

Reply:
changed